src/bundle/Resources/public/js/scripts/admin.search.autocomplete.js

```diff
@@ -0,0 +1,141 @@
+(function (global, doc, ibexa, Routing) {
+    const globalSearch = doc.querySelector('.ibexa-global-search');
+    const { getJsonFromResponse } = ibexa.helpers.request;
+    const { minQueryLength, resultLimit } = ibexa.adminUiConfig.suggestions;
+
+    if (!globalSearch) {
+        return;
+    }
+
+    const globalSearchInput = globalSearch.querySelector('.ibexa-global-search__input');
+    const clearBtn = globalSearch.querySelector(' .ibexa-input-text-wrapper__action-btn--clear');
+    const autocompleteNode = globalSearch.querySelector('.ibexa-global-search__autocomplete');
+    const autocompleteListNode = globalSearch.querySelector('.ibexa-global-search__autocomplete-list');
+    let controller;
```

Comment:
I think it should have more specific name, `controller` is really vast name, `searchAbortController` ir sth with autocomplete, maybe?